source/unified-test-format/tests/invalid/expectedCommandEvent-commandSucceededEvent-hasServerId-type.yml

```diff
@@ -0,0 +1,16 @@
+description: expectedEvent-commandSucceededEvent-hasServerId-type
+
+schemaVersion: '1.3'
+
+createEntities:
+  - client:
+      id: &client0 client0
+
+tests:
+  - description: invalid type for commandSucceededEvent hasServerId field
+    operations: []
+    expectEvents:
+      - client: *client0
+        events:
+          - commandSucceededEvent:
+              hasServerId: foo
```

Comment:
FYI: missing a newline at end of file. Doesn't make a difference, but the other files you created seem to have one.